### ISSUE TITLE
Update Cart with items display

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#2707]
![image](https://user-images.githubusercontent.com/2911176/45721963-a5e9b400-bbed-11e8-8e50-989e0926d806.png)

Zeplin: https://zpl.io/bAEMxv8

Based on current shopping cart functionality:
* Items between OFN header and checkout + total footer scrolls
* User can tap ‘cart’ in header to close cart
* Removed explicit ‘edit cart’ button from the design - @kirsten please let me know if you think this function is a must have.
* $XX is the total value for each product listing, removing the 1 x $100.00 = $100.00 version of product pricing
* Cart total + checkout button is fixed to bottom of viewport - #2718

**Comment by Matt-Yorkley** [comment]
On desktop there's buttons for both edit card and checkout. There's no buttons here for edit cart. I'd say it's pretty essential for mobile UX...

**Comment by kirstenalarsen** [comment]
this is the problem with the timelapse between design and dev . . I am pretty sure we discussed this but I cannot remember how we came to this decision. There is a google slide pack that has lots of the discussion and sometimes more info than zeplin, have a look at [slide  16](https://docs.google.com/presentation/d/10I8-3mRjxnPUXtEk04X3UERVPEPvqPjrEdF37_1KpmE/edit?pli=1#slide=id.g40fa39fcf7_0_0) . . where Yuko says she removed it and I didn't respond saying put it back! 

Would a text link for Edit Cart in the top menu, between '54 Items in your cart' and 'Close' do the trick? 

I will post a message in slack to start discussion of if/how we reengage Yuko to work with us as we're hitting need for adjustments . .
